js/common/modules/@arangodb/util.js

```diff
@@ -111,7 +109,11 @@ exports.codeFrame = function (e, basePath, withColor = internal.COLOR_OUTPUT) {
       const location = `@ ${
         basePath ? ctx.fileName.slice(basePath.length + 1) : ctx.fileName
       }:${ctx.lineNumber}:${ctx.columnNumber}\n`;
-      return (withColor ? chalk.grey(location) : location) + frame;
+      return (
+        withColor
+        ? `${internal.COLORS.COLOR_BRIGHT}${location}${internal.COLORS.COLOR_RESET}`
```

Comment:
I'm not sure what our equivalent of bright black (i.e. grey) is but I assume this is it?